app/controllers/peeps.rb

```diff
@@ -0,0 +1,36 @@
+class App < Sinatra::Base
+  get '/' do
+    redirect '/peeps'
+  end
+
+  get '/peeps' do
+    @user = current_user(session[:user_id])
+    @peeps = Peep.all(:order => [ :created_at.desc ])
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)